test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -62,7 +62,21 @@ def setup
       ignore_cvv: 'true',
       commerce_indicator: 'internet',
       user_po: 'ABC123',
-      taxable: true
+      taxable: true,
+      sales_slip_number: '456',
+      airline_agent_code: '7Q',
+      tax_management_indicator: 1,
+      installment_grace_period_duration: '1',
+      invoice_amount: '3',
+      original_amount: '4',
+      reference_data_code: 'ABC123',
+      invoice_number: '123',
+      mobile_remote_payment_type: 'A1',
+      vat_tax_rate: '1'
+    }
```

Comment:
I didn't end up writing tests for each individual field, bc it seemed more important to make sure they'd be added to the request in order no matter the transaction type so I added them to the main options hash. I did include a few remote tests or addition where the situation seemed to call for it, like adding the `capture_options` since that field would only be added to a capture request.

Reply:
I think that's reasonable; given the strictness of the XML parser errors that you fought with, if this options hash doesn't cause problems throughout the test suite, we can assume the fields are applied safely to the request